src/main/java/de/dennisguse/opentracks/util/RecyclerViewSwipeDeleteCallback.java

```diff
@@ -0,0 +1,90 @@
+package de.dennisguse.opentracks.util;
+
+import android.content.Context;
+import android.graphics.Canvas;
+import android.graphics.Paint;
+import android.graphics.PorterDuff;
+import android.graphics.PorterDuffXfermode;
+import android.graphics.drawable.ColorDrawable;
+import android.graphics.drawable.Drawable;
+import android.view.View;
+
+import androidx.annotation.NonNull;
+import androidx.core.content.ContextCompat;
+import androidx.recyclerview.widget.ItemTouchHelper;
+import androidx.recyclerview.widget.RecyclerView;
+
+import de.dennisguse.opentracks.R;
+
+abstract public class RecyclerViewSwipeDeleteCallback extends ItemTouchHelper.Callback {
+
+    Context context;
```

Comment:
protected?

Reply:
In fact, we don't need the context like a class member (only a local variable).